### ISSUE TITLE
Build in AS7.0

### ISSUE THREAD

**Issue by prenticedavid** [GoergPflug/AttinyStreamGfxApi#1]
I created an AS7.0 Solution.
Created a /dev directory and copied tiny_multi_os.h to it
Created Project: 3d_engine
Added 3d_engine.c
Added F_CPU=8000000 to Symbols
Added /dev to the Directories.

AS7.0 will compile but the SSD1306 does not work.  i.e. blank screen.

Tiny85, SSD1306 is connected to PB0, PB2, GND, VCC on a STK500.
Some projects expect "userfont.h" but there is no explanation of what format font you require.

David.

**Comment by GoergPflug** [comment]
the only project using userfont is acutally "flexgrid", i forgot to remove the defines, normally everything resides in a single directory, i will fix that...(by just removing "#define enable_userfont" in al examples except flexgrid)

 if it is not working your display might have a different adress, change

#define SSD1306_ADDRESS 0x78
in tiny_multi_os , probably to 0x7a

all demos are running at 16mhz on my attiny.

thank you for the feedback.

**Comment by prenticedavid** [comment]
Thanks for the updated files.  The examples build and run on the 8MHz Tiny85 now.

However `arduino_text_scroll_example_using_wire.ino` does not run on a Uno.
I added a ./dev directory to the sketch folder.   Changed the includes to use `<./dev`> instead of `</dev>`
It builds and uploads but the SSD1306 remains black.

You would normally put local header files in the sketch folder.   And `#include "header.h"`

David.
